pysiaf/tests/test_miri.py

```diff
@@ -30,7 +30,7 @@
 sys.path.append(test_data_dir)
 import mirim_siaf_testdata
 
-
+@pytest.mark.xfail
```

Comment:
I think the necessary `import pytest` is missing

Reply:
Done! I must have missed commiting it somehow